devices/kintexu/familyInfo.xml

```diff
@@ -0,0 +1,5874 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<device_description>
+	<family>kintexu</family>
+	<site_types>
+		<site_type>
+			<name>SLICEL</name>
+			<is_slice/>
+			<compatible_types>
+				<compatible_type>SLICEM</compatible_type>
+			</compatible_types>
+			<bels>
+				<bel>
+					<name>A5LUT</name>
+					<type>SLICEL_A5LUT</type>
```

Comment:
Yes, I agree. Updating the BEL types is a TODO item (we don't even have support for them yet in RapidSmith yet).